conformance/base/manifests.yaml

```diff
@@ -223,6 +223,42 @@ spec:
 ---
 apiVersion: v1
 kind: Service
+metadata:
+  name: infra-backend-delay
+  namespace: gateway-conformance-infra
+spec:
+  selector:
+    app: infra-backend-delay
+  ports:
+  - protocol: HTTP
```

Comment:
@frankbu It looks like this should be `TCP` - conformance tests fail with 

```
Service "infra-backend-delay" is invalid: spec.ports[0].protocol: Unsupported value: "HTTP": supported values: "SCTP", "TCP", "UDP"
```

Reply:
if this is set to `TCP` then tests are passing :

```bash
❯ go test ./conformance/conformance_test.go -args -gateway-class=istio -supported-features=Mesh --debug=true
ok  	command-line-arguments	51.791s
```

Reply:
Fixed. Thanks for testing it @SRodi!